ci/scripts/utils/ci_utils.sh

```diff
@@ -155,3 +155,25 @@ function publish_logs() {
     fi
     echo "${URL}"
 }
+
+function cleanup_experiment() {
+    # cleanup_experiment function
+    # This function takes a directory path of an EXPDIR and
+    # gets HOMEDIR value from config.base to
+    # clean up the archived files on disk.
+
+    local PSLOT_PATH
+    local pslot
+    local HOMEDIR
+    local ARCHIVEDIR
+
+    PSLOT_PATH="$1"
+    pslot=$(basename "${PSLOT_PATH}")
+    HOMEDIR=$(grep 'export HOMEDIR=' "${PSLOT_PATH}/config.base" | cut -d'=' -f2 | tr -d '[:space:]"' || true) || true
+    eval HOMEDIR="${HOMEDIR}"
```

Comment:
Try this
```suggestion
    local regex="^export HOMEDIR=(.*)"
    local lines
    mapfile -t lines < "${PSLOT_PATH}/config.base"
    local line
    for line in "${lines[@]}"; do
        if [[ "${line}" =~ ${regex} ]]; then
            HOMEDIR=$(echo "${BASH_REMATCH[1]}" | envsubst)
        fi
    done
```

Reply:
Wow looks strong,  yeah I'll try that.

Reply:
@DavidHuber-NOAA These archive paths designations should not be done with these one off regex injections.  Do we have a central location with an aggregation of known archival points?

Reply:
I noticed that `envsubst` actually worked to resolve **$USER** without using `eval`.

Reply:
@WalterKolczynski-NOAA `envsubst` was able to resolve **$USER** in **$HOMDIR** without the use of `eval`.